src/parcel_snoopl.jl

```diff
@@ -0,0 +1,49 @@
+"""
+    times, info = SnoopCompile.read_snoopl("func_names.csv", "llvm_timings.yaml")
+
+Reads the log file produced by the compiler and returns the structured representations.
+"""
+function read_snoopl(func_csv_file, llvm_yaml_file; tmin_secs=0.0)
+    func_csv = CSV.File(func_csv_file, header=false, delim='\t', types=[String, String])
+    llvm_yaml = YAML.load_file(llvm_yaml_file)
+
+    jl_names = Dict(r[1] => r[2] for r in func_csv)
```

Comment:
Not tested, but I'm guessing this won't be inferrable (and after my invalidations work I'm more sensitive to this than is probably warranted, so feel free to ignore this). You could optionally consider
```suggestion
    jl_names = Dict(r[1]::String => r[2]::String for r in func_csv)
```
or using `Dict{String,String}(...)`. The former has the slight advantage of applying the bandaid at the earliest possible moment and therefore providing the greatest resistance to invalidation.

Likewise, you could optionally annotate the type of `llvm_yaml` since `YAML.load_file` seems to be uninferrable. (Note it supports a `dicttype` keyword.)

Reply:
Thanks great suggestion. I'll commit your suggestion! :)